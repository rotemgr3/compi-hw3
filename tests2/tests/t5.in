//piazza 68 - should be ok (notice y) +
override int foo(int a) {return 5;}
override int foo(int a, int y) {return 5;}
void main() {
	int y = foo(5);
	int z = foo(5,7);
	return;
}
