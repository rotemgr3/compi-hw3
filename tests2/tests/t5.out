---end scope---
a INT -1
---end scope---
a INT -1
y INT -2
---end scope---
y INT 0
z INT 1
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
foo (INT)->INT 0
foo (INT,INT)->INT 0
main ()->VOID 0
